pdns/recursordist/aggressive_nsec.hh

```diff
@@ -48,6 +48,11 @@ public:
   {
   }
 
+  void setMaxEntries(uint64_t number)
+  {
+    d_maxEntries = number;
```

Comment:
Would it make sense to turn `d_maxEntries` into a `std::atomic<uint64_t>`, if only to make Thread Sanitizer happy?